### ISSUE TITLE
The new "Associate common subtitle files with Subtitle Edit" option during 3.6.5 install creates file associations all with the same wrong string: `C:\Program Files\Subtitle Edit\`

### ISSUE THREAD

**Issue by darkred** [SubtitleEdit/subtitleedit#5851]
Using Subtitle Edit on win11 x64.
I had SE 3.6.4 installed. That offered an `Associate SubRip Text (.srt) subtitle files with Subtitle Edit` checkbox during install.
I updated to 3.6.5. This version offers an `Associate common subtitle files with Subtitle Edit` checkbox instead. I ticked it and after install I noticed that all file associations created, have the same wrong string: `C:\Program Files\Subtitle Edit\`
Here is a screenshot example from File Explorer:
![2022-03-23_013555](https://user-images.githubusercontent.com/723651/159593486-913decd1-5663-48d5-918d-d8e10c0b57d9.jpg)
and here a screenshot from Nirsoft's FileTypesMan utility showing all the file associations created:
![2022-03-23_013106](https://user-images.githubusercontent.com/723651/159593041-2a1adabd-51ba-486d-a576-d95f51454743.jpg)

Thank you

**Comment by niksedk** [comment]
Thx for the info :)

Is this better? https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.5/SubtitleEditBetaSetup.zip

**Comment by darkred** [comment]
Thanks for the quick response. I updated to that beta version, and then killed + restarted explorer.exe and even rebooted, but unfortunately the issue remains:
![2022-03-23_194531](https://user-images.githubusercontent.com/723651/159763265-51d1252f-9ae7-4002-a124-48a4b94b8b55.jpg)

![2022-03-23_194611](https://user-images.githubusercontent.com/723651/159763274-d592c2a9-8c5c-4808-8483-d57f0b7d696b.jpg)

Notice that the type now also has a `Subtitle.exe` in its end.

**Comment by niksedk** [comment]
OK, I don't understand the issue...  what do you expect the values to be?

**Comment by darkred** [comment]
This is how it was when I had 3.6.4 (or uninstalling 3.6.5 and installing 3.6.4):
![2022-03-24_002947](https://user-images.githubusercontent.com/723651/159806780-c0e91c28-2dd8-438c-a7ae-9bf7f5e23e23.jpg)

With 3.6.5 stable:
![159593486-913decd1-5663-48d5-918d-d8e10c0b57d9 (1)~](https://user-images.githubusercontent.com/723651/159808600-d57833ea-62f2-49e3-9033-3638c1454679.jpg)

and with your 3.6.5 beta:
![](https://user-images.githubusercontent.com/723651/159763265-51d1252f-9ae7-4002-a124-48a4b94b8b55.jpg)

&nbsp;
In other words after 3.6.5 even the existing .srt description ("SubRip Text subtitles") becomes lost.

The new, global `C:\Program Files\Subtitle Edit\SubtitleEdit.exe` description seems invalid/irrelevant to me.

The expected descriptions in the Type column for me should be (based on this [wiki page](https://en.wikipedia.org/wiki/Subtitles)):
```
SAMI subtitles
SubRip subtitles
VobSub subtitles
```


*(the `Text` in `SubRip Text subtitles` could be omitted)*

**Comment by in4u** [comment]
@niksedk Just updated to 3.6.5 and ran into this exact same problem.

I think there's an unwanted `space` character in the file extension mapping, e.g. it should've been `SubtitleEdit` instead of `Subtitle Edit`. Like, in the FileTypesMan screenshot shared by @darkred, it can be seen that `.srt` is mapping to `Subtitle Edit.srt` instead of `SubtitleEdit.srt`. In my system's registry, `HKCR\SubtitleEdit.srt` has the correct description value `SubRip Text subtitles` while `HKCR\Subtitle Edit.srt` has the default description value `C:\Program Files\Subtitle Edit\`.

So, my guess is that either you need to remove the extraneous `space` character, or if the space is intentionally added in this update then the relevant extension descriptions should also be added accordingly.

**Comment by BKR-TN** [comment]
Here is corrected values (I hope the names is correct) in both, Windows Registry and Inno Setup formats:

### Windows Registry

```
Windows Registry Editor Version 5.00

[HKEY_CLASSES_ROOT\Subtitle Edit.ass]
@="Advanced SubStation Alpha Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.dfxp]
@="Distribution Format Exchange Profile Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.itt]
@="iTunes Text Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.sbv]
@="SBV Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.smi]
@="SMI Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.srt]
@="SubRip Text Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.ssa]
@="SubStation Alpha Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.stl]
@="STL Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.sub]
@="VobSub/MicroDVD Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.sup]
@="Blu-ray PGS Subtitles"

[HKEY_CLASSES_ROOT\Subtitle Edit.vtt]
@="Web Video Text Tracks (WebVTT) Subtitles"
```
### Inno Setup

```
[Registry]
Root: HKCR; SubKey: Subtitle Edit.ass; ValueType: string; ValueData: Advanced SubStation Alpha Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.dfxp; ValueType: string; ValueData: Distribution Format Exchange Profile Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.itt; ValueType: string; ValueData: iTunes Text Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.sbv; ValueType: string; ValueData: SBV Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.smi; ValueType: string; ValueData: SMI Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.srt; ValueType: string; ValueData: SubRip Text Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.ssa; ValueType: string; ValueData: SubStation Alpha Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.stl; ValueType: string; ValueData: STL Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.sub; ValueType: string; ValueData: VobSub/MicroDVD Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.sup; ValueType: string; ValueData: Blu-ray PGS Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
Root: HKCR; SubKey: Subtitle Edit.vtt; ValueType: string; ValueData: Web Video Text Tracks (WebVTT) Subtitles; Flags: uninsdeletevalue uninsdeletekeyifempty
```

**Comment by niksedk** [comment]
OK, I've tried to update the installer: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.5/SubtitleEditBetaSetup.zip

Changes: https://github.com/SubtitleEdit/subtitleedit/commit/83ceb746394bf9980165619b89aeb7e78f679c41

**Comment by darkred** [comment]
I tried your latest beta (after uninstalling previous version) and unfortunately, only the .srt file type is fixed:
![2022-03-31_191420](https://user-images.githubusercontent.com/723651/161102348-394743f5-5191-4d30-9fb6-315b3a93a122.jpg)
and in FileTypesMan all other types are now listed are completely empty:
![2022-03-31_191658](https://user-images.githubusercontent.com/723651/161102360-cf22bedc-6b28-4b5a-9286-ccc95cf26ea1.jpg)

**Comment by niksedk** [comment]
OK, hopefully this is better: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.5/SubtitleEditBetaSetup.zip

**Comment by darkred** [comment]
With the new beta all types are fixed. Thank you!
![2022-03-31_231416](https://user-images.githubusercontent.com/723651/161141657-8991da6a-8865-4f17-b705-9ec55a7225df.jpg)

**Comment by BKR-TN** [comment]
All things is good now, but there is a small typo in [line 459 in "Subtitle_Edit_installer.iss"](https://github.com/SubtitleEdit/subtitleedit/blob/master/installer/Subtitle_Edit_installer.iss#L459) file, you forget to add quotation mark at the end of the **`ValueData: "Advanced Sub Station Alpha subtitle file;`**<--here.

**Comment by United600** [comment]
Sorry for reopening this issue but I think that it's still a regression from the [previous "Subtitle_Edit_installer.iss"](https://github.com/SubtitleEdit/subtitleedit/blob/c6e9f1f3037e7d0cf89f3ae12a9c105d5755b1be/installer/Subtitle_Edit_installer.iss).

It's missing the important [FriendlyTypeName](https://docs.microsoft.com/en-us/windows/win32/shell/how-to-register-a-file-type-for-a-new-application#step-4) for localization and the [Default Files capabilites](https://docs.microsoft.com/en-us/windows/win32/shell/default-programs?redirectedfrom=MSDN#capabilities).

**Comment by niksedk** [comment]
@United600: More info or PR welcome :)
